Build.fsx

```diff
@@ -1,15 +1,43 @@
 ﻿#r @"packages/FAKE.Core/tools/FakeLib.dll"
 
 open Fake
+open Fake.AssemblyInfoFile
 open Fake.Testing
 open System
+open System.Diagnostics;
+open System.Text.RegularExpressions
 
 let releaseFolder = "Release"
 let nunitToolsFolder = "Packages/NUnit.Runners.2.6.2/tools"
 let nuGetOutputFolder = "NuGetPackages"
 let solutionsToBuild = !! "Src/*.sln"
 let signKeyPath = FullName "Src/AutoFixture.snk"
 
+type GitVersion = { apiVersion:string; nugetVersion:string; }
+let getGitVersion = 
```

Comment:
I like the idea of letting the version number be calculated based on the history. 👍
However, I strongly suggest we use an established tool like [GitVersion](https://github.com/GitTools/GitVersion) for that instead of rolling our own.